components/skus/common/features.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/skus/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace skus {
+namespace features {
+
+const base::Feature kSkusFeature {
+  "SkusFeature",
+#if defined(OS_IOS)
+      base::FEATURE_ENABLED_BY_DEFAULT
```

Comment:
this should not be enabled by default, we added this flag so we could merge this code and iterate

Reply:
iOS cannot change this flag at runtime currently, if we have it disabled by default we will also have to add checks to return nullptr in our factories as well and we won't be able to actually integrate with it in iOS side

Reply:
I guess we need to sort out some way for ios to enable flags at runtime, but please add a comment to that effect